src/css/_guest.scss

```diff
@@ -0,0 +1,50 @@
+.guest {
+  display: flex;
+  justify-content: center;
+}
+
+.guest-info{
+  color: black;
+}
+
+.guest-header {
+  font-size: 1.5em;
+}
+
+.guest-visit {
```

Comment:
You could probably implement a Mixin for this and guest-search since they're the same, It'll cut down some lines of code